tests/robustness/snapstore/snapstore.go

```diff
@@ -0,0 +1,22 @@
+// Package snapstore describes entities that can accept
+// arbitrary metadata and flush it to a persistent repository.
+package snapstore
```

Comment:
Does snapmeta, or snapmetadata make sense?
Or is the snap store expected to contain additional data other than snapshot metadata?

Reply:
Changed it to snapmeta. I suppose it could store additional data other than snapshot metadata, for instance logs of actions taken over the course of testing the repo. We can change the name if we feel "snapmeta" doesn't fit.